pkg/kmsg/README.md

```diff
@@ -0,0 +1,2 @@
+A simple program to log the kernel log buffer
```

Comment:
Can you drop this. While this is a one liner, I prefer to have package documentation in `./docs`. Main reason is that if people change the docs it affects the git tree hash, requiring package rebuild for no code changes. There are some other packages which have `README.md`s but that seem the minority.

Or add a comment in `main.go` after `package main`